runtime/doc/treesitter.txt

```diff
@@ -682,8 +682,7 @@ inspect_tree({opts})                           *vim.treesitter.inspect_tree()*
     Can also be shown with `:InspectTree`.                      *:InspectTree*
 
     Parameters: ~
-      • {opts}  (table|nil) Optional options table with the following possible
-                keys:
+      • {opts}  InspectTreeOpts ? Optional options table with the following possible keys:
```

Comment:
I'm not sure why this type wasn't public in the first place. If intended I can undo the change.

Reply:
Undoing the change since `doxygen` is broken and I cannot it namespace it.